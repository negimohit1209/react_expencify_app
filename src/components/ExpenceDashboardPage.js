import React from "react";
import ExpenseList from './ExpenseList';
import ExpenceListFilters from './ExpenceListFilters';
import ExpencesSummary from './ExpencesSummary';
const ExpenceDashboardPage = () => (
    <div>
        <ExpencesSummary />
        <ExpenceListFilters />
        <ExpenseList />
    </div>

)
export default ExpenceDashboardPage;