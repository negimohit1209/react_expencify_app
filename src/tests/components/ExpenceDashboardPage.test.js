import React from 'react';
import { shallow } from 'enzyme';
import ExpenceDashboardPage from '../../components/ExpenceDashboardPage';

test('should render ExpenceDashboardPage.', () => {
    const wrapper = shallow(<ExpenceDashboardPage />);
    expect(wrapper).toMatchSnapshot();
});