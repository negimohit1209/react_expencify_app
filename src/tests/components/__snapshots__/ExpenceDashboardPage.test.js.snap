// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenceDashboardPage. 1`] = `
<div>
  <Connect(ExpencesSummary) />
  <Connect(ExpenceListFilters) />
  <Connect(ExpenseList) />
</div>
`;
